src/AstGen.zig

```diff
@@ -1756,13 +1755,15 @@ fn structInitExpr(
                     var error_notes = std.ArrayList(u32).init(astgen.arena);
 
                     for (record.items[1..]) |duplicate| {
-                        try error_notes.append(try astgen.errNoteTok(duplicate, "other field here", .{}));
+                        try error_notes.append(try astgen.errNoteTok(duplicate, "duplicate name here", .{}));
                     }
 
+                    try error_notes.append(try astgen.errNoteNode(node, "struct declared here", .{}));
+
                     try astgen.appendErrorTokNotes(
                         record.items[0],
-                        "duplicate field",
-                        .{},
+                        "struct field name conflict: '{s}'",
```

Comment:
Assuming this wasn't specifically requested by a core team member for any reason: you really don't need to put the field name here. This information already exists: it's the token the error points to. Adding more words to the error message just makes it take longer to read.

Reply:
You're all good with the personal life stuff. I hope it went well (i believe you were moving?).

It wasn't requested by a core team member, and I agree, it is duplicate information. I'll remove it in the following commit, as well as update the tests.